src/executables/util_service.c

```diff
@@ -1836,6 +1855,38 @@ process_server (int command_type, int argc, char **argv, bool show_usage, bool c
   return status;
 }
 
+/*
+ * process_server_start_single_node - start both a page and a transaction server,
+ *                                     the page server must be started before transaction server
+ *
+ * return:
+ */
+static int
+process_server_start_single_node (char *server_name, int command_type)
+{
+  int status, pid;
+
+  const char *args_page[] = { UTIL_CUBRID_NAME, "-t", "page", server_name, NULL };
+  status = proc_execute (UTIL_CUBRID_NAME, args_page, false, false, false, &pid);
+
+  if (status == NO_ERROR && !is_server_running (CHECK_PAGE_SERVER, server_name, pid))
+    {
+      status = ER_GENERIC_ERROR;
+    }
+  print_result (PRINT_PAGE_SERVER_NAME, status, command_type);
```

Comment:
Stop if page server could not be started.
```suggestion
  print_result (PRINT_PAGE_SERVER_NAME, status, command_type);
  if (status != NO_ERROR)
    {
      return status;
    }
```